src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,140 @@
+package duke.storage;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+
+import duke.data.TaskList;
+import duke.data.exception.DukeException;
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Task;
+import duke.tasks.ToDo;
+
+/**
+ * This class encapsulates the Storage
```

Comment:
Perhaps you could give more description on what the class does, e.g. in this case "... which stores tasks into the local hard disk". 🤔 

Same for the other classes. 😄